tests/utils/test_embeddings.py

```diff
@@ -241,64 +252,65 @@ def testSubgraphEmbeddingFeasibilityFilter(self):
         )
         # Check tetrahedron cannot be embedded on a graph with a triangle + 0,1,2 connectivity nodes.
 
-    def testRasterBreadthSubgraphUpperBound(self):
+    def testGraphRowsSubgraphUpperBound(self):
         L = np.random.randint(2) + 2
         T = dnx.zephyr_graph(L - 1)
-        self.assertEqual(L - 1, raster_breadth_subgraph_upper_bound(T=T))
+        self.assertEqual(L - 1, graph_rows_upper_bound(T=T))
         T = dnx.pegasus_graph(L)
-        self.assertEqual(L, raster_breadth_subgraph_upper_bound(T=T))
+        self.assertEqual(L, graph_rows_upper_bound(T=T))
         T = dnx.chimera_graph(L, L - 1, 1)
-        self.assertEqual(L, raster_breadth_subgraph_upper_bound(T=T))
+        self.assertEqual(L, graph_rows_upper_bound(T=T))
 
-    def testRasterBreadthSubgraphLowerBound(self):
+    def testGraphRowsSubgraphLowerBound(self):
         L = np.random.randint(2) + 2
         T = dnx.zephyr_graph(L - 1)
-        self.assertEqual(L - 1, raster_breadth_subgraph_lower_bound(S=T, T=T))
+        self.assertEqual(L - 1, graph_rows_lower_bound(S=T, T=T, one_to_one=True))
         self.assertEqual(
-            L - 1, raster_breadth_subgraph_lower_bound(S=T, topology="zephyr")
+            L - 1, graph_rows_lower_bound(S=T, topology="zephyr", one_to_one=True)
         )
         T = dnx.pegasus_graph(L)
-        self.assertEqual(L, raster_breadth_subgraph_lower_bound(S=T, T=T))
+        self.assertEqual(L, graph_rows_lower_bound(S=T, T=T, one_to_one=True))
         self.assertEqual(
-            L, raster_breadth_subgraph_lower_bound(S=T, topology="pegasus")
+            L, graph_rows_lower_bound(S=T, topology="pegasus", one_to_one=True)
         )
         T = dnx.chimera_graph(L, L - 1, 1)
-        self.assertEqual(L, raster_breadth_subgraph_lower_bound(S=T, T=T))
+        self.assertEqual(L, graph_rows_lower_bound(S=T, T=T, one_to_one=True))
         self.assertEqual(
-            L, raster_breadth_subgraph_lower_bound(S=T, topology="chimera", t=1)
+            L, graph_rows_lower_bound(S=T, topology="chimera", t=1, one_to_one=True)
         )
 
         m = 6
         S = dnx.chimera_graph(m)  # Embeds onto Zephyr[m//2]
         self.assertEqual(
-            m // 2, raster_breadth_subgraph_lower_bound(S=S, topology="zephyr")
+            m // 2, graph_rows_lower_bound(S=S, topology="zephyr", one_to_one=True)
         )
         T = dnx.zephyr_graph(m)
-        self.assertEqual(m // 2, raster_breadth_subgraph_lower_bound(S=S, T=T))
+        self.assertEqual(m // 2, graph_rows_lower_bound(S=S, T=T, one_to_one=True))
 
-    def testRasterEmbeddingSearchBasic(self):
+    def testFindSublatticeEmbeddingsBasic(self):
```

Comment:
should these tests account for edge cases? e.g., invalid input

Reply:
We need to add self.assertRaises() tests, but beyond that I'm not sure we need to check every single wrong type throws an error given that we are type hinting